build.gradle

```diff
@@ -1,39 +1,25 @@
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
-    id "com.exactpro.th2.gradle.base" version "0.1.1"
-    id "com.exactpro.th2.gradle.publish" version "0.1.1"
-    id "org.jetbrains.kotlin.jvm" version "$kotlin_version"
-    id 'org.jetbrains.kotlin.kapt' version "$kotlin_version"
+    alias(libs.plugins.th2.publish)
+    alias(libs.plugins.kotlin.jvm)
+    alias(libs.plugins.kotlin.kapt)
+    alias(libs.plugins.protobuf)
+    alias(libs.plugins.jmh)
     id "java-library"
     id "java-test-fixtures"
     id "maven-publish"
-    id "com.google.protobuf" version "0.9.4"
-    id "me.champeau.jmh" version "0.7.2"
 }
 
 group = 'com.exactpro.th2'
 version = release_version
 
 ext {
-    grpcVersion = '1.62.2'
-    protobufVersion = '3.25.3'
-    serviceGeneratorVersion = '3.6.1'
-
-    cradleVersion = '5.4.1-dev'
-    junitVersion = '5.10.3'
-
-    jmhVersion = '1.37'
-    autoValueVersion = '1.10.4'
     genBaseDir = file("${buildDir}/generated/source/proto")
```

Comment:
note: using of `buildDir` is deprecated. We should use `project.layout.buildDirectory.file` instead in this case